contrib/cirrus/runner.sh

```diff
@@ -62,7 +62,7 @@ _run_cross() {
 }
 
 _run_validate() {
-    podmanmake validate-local BUILDTAGS="$BUILDTAGS"
+    podmanmake validate-local validate-docs BUILDTAGS="$BUILDTAGS"
```

Comment:
This works but I’m worried that it will be confusing.

Maybe name this `…/runner.sh validate-and-validate-docs`, something that doesn’t directly correspond to a `Makefile` rule.